### ISSUE TITLE
[Wii] GBA-GC Link Cable Support

### ISSUE THREAD

**Issue by GB-CG** [mgba-emu/mgba#200]
I'll be nice to try using a GBA as a controller for the Wii port, although that sounds like a tough one to do.

**Comment by endrift** [comment]
Cool idea, but it's a ways off before I do this.

**Comment by nateify** [comment]
If this is of any help: https://github.com/FIX94/gc-gba-link-cable-demo

**Comment by GB-CG** [comment]
@nateify Yeah that's really cool, but unfortunately I'm not a coder so there's not much I can do.

**Comment by endrift** [comment]
I saw this, it looks tremendously useful.

**Comment by Arlisbloxer05** [comment]
any updates on this?

**Comment by Whovian9369** [comment]
Is this closable as Dolphin introduced in-Dolphin support for (clock-matched) ability?

**Comment by endrift** [comment]
This actually refers to using a real GBA to control the Wii port of mGBA, not a GBA-GC interaction like Dolphin has (and mGBA has had for a few months before that too).

**Comment by Whovian9369** [comment]
I completely missed that, apologies!

**Comment by EthanMac1915** [comment]
Hey, has this ever been implemented? Picked up a gba link cable recently and wanted to know if it was usable on my wii with some gba games

**Comment by Extrems** [comment]
There's a standalone ROM in the extra package. It's even available in e-Reader dot code format.
